components/Cases/CaseRecap/CaseRecap.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import Spinner from 'components/Spinner/Spinner';
+import Summary from 'components/Summary/Summary';
+import { formatDate } from 'utils/date';
+import { useCaseByResident } from 'utils/api/cases';
+import * as form from 'data/forms';
+
+const CaseRecap = ({ personId, recordId }) => {
+  const { data: record, error: recordError } = useCaseByResident(
+    personId,
+    recordId
+  );
+
+  const recordData =
+    record && record.caseFormData && record.caseFormData.form_name_overall;
+  const fileData = form[recordData];
+
+  if (recordError || (recordData && !fileData)) {
+    return <ErrorMessage />;
+  }
+  if (!record || !fileData) {
+    return <Spinner />;
+  }
+  return (
+    <>
+      <h3 className="govuk-heading-m govuk-custom-text-color govuk-!-margin-bottom-0">
+        FORM CREATION
+      </h3>
+      <dl className="govuk-summary-list">
+        <div className="govuk-summary-list__row">
+          <dt className="govuk-summary-list__key">Date created</dt>
+          <dd className="govuk-summary-list__value">
+            {formatDate(record.caseFormTimestamp)}
```

Comment:
This should be `dateOfEvent` and if it's not present `caseFormTimestamp`

Reply:
and I think we want the 2 formats consistent <img width="595" alt="Screenshot 2021-03-08 at 11 00 13" src="https://user-images.githubusercontent.com/435399/110312801-ddd75280-8005-11eb-8f2c-b90bf8ac02e4.png">